evm_arithmetization/src/cpu/kernel/interpreter.rs

```diff
@@ -547,6 +548,20 @@ impl<F: Field> State<F> for Interpreter<F> {
 
         self.perform_state_op(op, row)
     }
+
+    fn log_debug(&self, msg: String) {
+        if !self.is_jumpdest_analysis {
+            log::debug!("{}", msg);
+        }
+    }
+
+    fn log_info(&self, _msg: String) {}
```

Comment:
is the intent to not print anything, or to rely on the default trait impl? If the latter, it should be removed.

Reply:
(Note that there is no `info` logs during kernel execution, so we can probably rely on the default impl anyway, it won't change anything.

Reply:
(Note that there is no `info` logs during kernel execution, so we can probably rely on the default impl anyway, it won't change anything.

Reply:
It was the first, but it is indeed useless here and we can just rely on the default implementation.